app.ts

```diff
@@ -0,0 +1,2 @@
+let message: string = 'Wanna Play?';
```

Comment:
```suggestion
const message = 'Wanna Play?';
```

`let` is a mutable variable. We prefer `const` except when mutation is actually needed.

Writing the type explicitly is OK, but we usually skip it unless it's interesting and useful to read.